### DISCUSSION TITLE
Changing Layer Visible Status

### DISCUSSION THREAD

**Discussion by silentdevnull** [dlemstra/Magick.NET#1300] [Help]
Hello, 

Is there away to change the layers visible in a PSD file then save it back as a PSD file with a new name? 

Thank you,

**Comment by dlemstra** [comment]
You can hide a layer by setting `image.Compose` to `CompositeOperator.No` and you can make it visible by setting it to `CompositeOperator.Undefined`. But you will loose some of the information in the PSD file because this library cannot read everything of the PSD file.
